install.rdf

```diff
@@ -5,20 +5,19 @@
     <em:bootstrap>true</em:bootstrap>
     <em:type>2</em:type>
     <em:unpack>false</em:unpack>
-    <em:version>1.0.0</em:version>
+    <em:version>0.1.2</em:version>
     <em:name>Universal Search Addon</em:name>
     <em:description>Universal Search desktop FF experiments in addon format</em:description>
     <em:creator/>
     <em:homepageURL>https://github.com/mozilla/universal-search</em:homepageURL>
     <em:optionsURL>chrome://universalsearch/content/preferences.xul</em:optionsURL>
     <em:optionsType>2</em:optionsType>
-    <!-- TODO: figure out the release URL (#24) -->
-    <em:updateLink>https://example.com/dummy.xpi</em:updateLink>
+    <em:updateURL>https://s3-us-west-2.amazonaws.com/universal-search/universal-search.xpi</em:updateURL>
     <em:targetApplication>
       <Description>
         <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
-        <em:minVersion>38.0</em:minVersion>
-        <em:maxVersion>54.0</em:maxVersion>
```

Comment:
Yeah, sounds good. I thought a '*' maxVersion would cause validation errors, so that you had to go by the accepted versions [on this page](https://addons.mozilla.org/en-US/firefox/pages/appversions/), currently 49.0 is the max. But if this works, great.